minerl/herobraine/hero/handlers/agent/actions/keyboard.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, command, *keys):
         self.keys = keys
 
     def from_universal(self, x):
-        actions_mapped = set(str(k) for k in x['custom_action']['actions']['keys'])
+        actions_mapped = list(x['custom_action']['actions'].keys())
```

Comment:
@pzhokhov Can you explain this change? Seems like we should keep set()

Reply:
tbh don't remember too well; there may have been an issue with repeated keys... The end result of human level input did not use this handler, so I suspect the change was not necessary.

Reply:
Used the set notation for now